% Chapter Template

\chapter[Publication 3]{Publication 3} % Main chapter title

\label{ChapterX} % Change X to a consecutive number; for referencing this chapter elsewhere, use \ref{ChapterX}
\section*{The effect of latitudinal light regime on seasonal gene expression in Antarctic krill (\textit{Euphausia superba})}

F. Höring, K. Michael, F. Piccolin, A. Biscontin, S. Kawaguchi, B. Meyer

In preperation for Marine Ecology Progress Series
%----------------------------------------------------------------------------------------
%	Abstract
%----------------------------------------------------------------------------------------

\section{Abstract}

Antarctic krill, \textit{Euphausia superba}, flexibly adapts its seasonal life
cycle to the highly variable conditions in the different latitudinal habitats
in the Southern Ocean. This study investigated how different latitudinal light
regimes affected seasonal gene expression in Antarctic krill. We analysed the
expression of various metabolic and regulatory genes in a controlled long-term
laboratory experiment under constant food supply simulating the latitudinal
light regimes 52$^{\circ}$S, 66$^{\circ}$S, and constant darkness as reference.
We found that latitudinal light regime induced seasonally rhythmic patterns of
gene expression that were more distinct under the high-latitude light regime
66$^{\circ}$S and were mostly disrupted under constant darkness. The simulated
light regimes clearly affected the expression patterns of genes related to
carbohydrate, energy, and lipid metabolism, lipid transport, translation, the
biosynthesis of signalling molecules as well as to the circadian clock. These
results suggest that Antarctic krill is able to adjust its seasonal gene
expression patterns and associated metabolic and regulatory processes to
different latitudinal light regimes. We propose that genes related to the
circadian clock, as well as prostaglandin, methyl farnesoate, neuropeptide
biosynthesis and receptor signalling may play a role in the regulation of
seasonal rhythms in Antarctic krill.

%-----------------------------------
%	INTRODUCTION
%-----------------------------------
\section{Introduction}

Antarctic krill, \textit{Euphausia superba}, holds a pivotal position in the
Southern Ocean food web forming a major link between primary production and
higher trophic levels such as fish, penguins, seals and whales. In the Atlantic
sector of the Southern Ocean, krill distribution extends to a remarkably broad
latitudinal range from ~51$^{\circ}$S to ~70$^{\circ}$S where it is exposed to
strong seasonality with extreme shifts in photoperiod, primary productivity and
sea ice extent \citep{quetin_behavioral_1991}.

Our knowledge on the molecular mechanisms that underlie Antarctic krill's
flexible behaviour in different latitudinal regions still remains fragmentary.
A better understanding of the regulatory and metabolic processes that govern
seasonal rhythms in Antarctic krill may improve the explanatory power of
Antarctic krill models. This is especially relevant under the recently observed
changes in sea ice extent and macrozooplankton distribution in the South West
Atlantic Sector \citep{atkinson_sardine_2014, atkinson_krill_2019,
steinberg_long-term_2015} that point to fundamental changes in the Southern
Ocean food web.

In the field, Antarctic krill shows pronounced seasonal rhythms of growth,
lipid metabolism, metabolic activity and maturity \citep{kawaguchi_krill_2007,
meyer_seasonal_2010}. Survival during periods of near-constant darkness is
accomplished by different overwintering strategies such as metabolic
depression, regressed development and growth and the utilisation of body lipid
stores \citep{meyer_overwintering_2012}. Regional differences in the timing of
reproduction \citep{spiridonov_spatial_1995}, growth
\citep{kawaguchi_modelling_2006}, gene expression \citep{seear_seasonal_2012},
and winter lipid storage and feeding activity \citep{schmidt_feeding_2014} have
also been observed in Antarctic krill.

Interestingly, winter krill from the low latitude region around South Georgia
(54$^{\circ}$S) have been found to differ the most, in terms of feeding
activity, body lipid storage and gene expression from higher latitude krill
\citep[and H{\"o}ring et al., in prep(Publication 1)]{schmidt_feeding_2014}. At
South Georgia, Antarctic krill was found to have higher feeding activities and
lower lipid stores in winter \citep{schmidt_feeding_2014, seear_seasonal_2012}
Moreover, seasonal differences in gene expression between summer and winter
were less pronounced in Antarctic krill from South Georgia region indicating
that Antarctic krill does not enter a distinct period of rest in this region in
winter (Höring et al., Publication 1). The flexible seasonal behaviour of Antarctic
krill at South Georgia may be an adaptation to the low-latitude light regime
which enhances food availability in that region year-round.

Recently, a long-term laboratory study has shown that Antarctic krill possesses
a flexible endogenous timing system, likely entrained by the prevailing light
regime \citep{horing_light_2018}. It seems to affect krill's seasonal response,
controlling multiple seasonal processes such as growth, feeding, lipid
metabolism, maturity, oxygen consumption and metabolic activity
\citep{horing_light_2018, piccolin_seasonal_2018}.

It has been suggested that Antarctic krill's circadian clock may play a role in
the timing of seasonal life cycle events \citep{piccolin_seasonal_2018}. The
circadian clock system has been functionally characterized in Antarctic krill
and includes a light mediated entraining mechanism
\citep{biscontin_functional_2017}. It may not only be important for the
synchronisation of daily rhythms of metabolic activity, gene expression
\citep{de_pitta_antarctic_2013, teschke_circadian_2011}, and diel vertical
migration \citep{gaten_is_2008}, but also for the measurement of seasonal day
length. Light conditions in early autumn may be critical for seasonal
entrainment in Antarctic krill as an upregulation of genes related to the
circadian clock and circadian-related opsins were observed in photoperiodic
controlled laboratory experiments during this time of the year
\citep{piccolin_seasonal_2018}.

This study complements the results from a recent two-year lab experiment,
analysing the effect of different latitudinal light regimes on the seasonal
cycle of Antarctic krill \citep{horing_light_2018}. \citet{horing_light_2018}
focussed on morphometric and lipid content data, whereas data on gene
expression and metabolic pathways were still missing. How different latitudinal
light regimes affect the molecular processes during the seasonal cycle of
Antarctic krill is still not yet clear.

The goal of this study was to characterize molecular pathways that contribute
to different seasonal responses of Antarctic krill under the simulated
latitudinal light regimes 52$^{\circ}$S and 66$^{\circ}$S, as well as constant
darkness (DD), under constant food supply. A set of genes known to be involved
in seasonal timing, reproduction, feeding, development, lipid, energy and
carbohydrate metabolism was analysed for (i) seasonal rhythmicity of gene
expression under the treatments 52$^{\circ}$S, 66$^{\circ}$S and DD over one
year, and (ii) differences in gene expression patterns between the three
treatments in seven months within a one year seasonal light cycle (April, June,
August, October, December, February and April the next year).


%-----------------------------------
%	MATERIALS AND METHODS
%-----------------------------------
\section{Materials \& Methods}
\subsection{Long-term lab experiments at the Australian Antarctic Division}

To test the effect of different light regimes on Antarctic krill, long-term lab
experiments were performed under constant food supply at the Australian
Antarctic Division aquarium in Hobart, Australia, over a period of two years
(January 2015 to December 2016). Three different light regimes were simulated:
1) natural light regime at 52$^{\circ}$S, 2) natural light regime at
66$^{\circ}$S, and 3) constant darkness (DD). The detailed experimental
conditions including details on krill sampling, experimental set-up, the
simulated light regimes and food supply can be found in an earlier study
investigating morphometric and lipid content data from the same experiment
\citep{horing_light_2018}. Frozen krill samples from the experiments were sent
to the Alfred-Wegener-Institute, Bremerhaven, for molecular analysis and stored
at \SI{-80}{\celsius}. Overall 126 individuals from three tanks/treatments
(52$^{\circ}$S, 66$^{\circ}$S, DD) were analysed including seven time points
per treatment (April 2015 to April 2016) with six biological replicates per
time point (Table I). 

%%% TODO Table I

\subsection{Sample processing and preparation of RNA and cDNA}

Krill heads were cut on dry ice and transferred to
RNA\textit{later}\texttrademark-ICE (Thermo Fisher Scientific, Waltham, MA,
USA) according to manual instructions. The heads were dissected using the
stereomicroscope Leica M125 C (Wetzlar, Germany) and a cooling element,
adjusted to \SI{4}{\celsius} by a Minichiller (Huber, Offenburg, Germany).
Loose parts such as thoracopods, the pigmented part of the eyes, and the front
part of the antennas were removed from the krill heads. The homogenization of
the dissected krill heads was carried out in
Precellys\textsuperscript{\textregistered} tubes (CKMix Tissue Homogenizing
Kit, Bertin corp., Rockville, MD, USA) containing \SI{600}{\micro\liter} TRIzol\texttrademark
Reagent per tube and sample (Thermo Fisher Scientific) using the
Precellys\textsuperscript{\textregistered} homogenisator with the
Cryolys\textsuperscript{\textregistered} cooling system (Bertin corp.) for 2*15
s at 5000 rpm at \SI{4}{\celsius}. After 5 min of incubation at room temperature, a phase
separation step with chloroform was carried out according to the TRIzol
protocol and \SI{200}{\micro\liter} of aqueous phase per sample were transferred to a new tube.
RNA was then purified with the Direct-zol\texttrademark RNA MiniPrep Plus kit
(Zymo Research Europe, Freiburg, Germany) including an on-column DNase I
treatment step, following manual instructions. RNA was eluted in \SI{50}{\micro\liter} of
RNase-free water and stored at \SI{-80}{\celsius}.  Quality and quantity of the RNA were
validated using a NanoDrop\texttrademark2000 UV-Vis Spectrophotometer (Thermo Fisher
Scientific) and the Agilent 2100 Bioanalyzer system (Agilent technologies,
Santa Clara, CA, USA).  cDNA was synthesized in \SI{50}{\micro\liter} reaction volume with
final concentrations of \SI{4}{\micro\gram} RNA, 100U RevertAid\texttrademark H Minus Reverse
Transcriptase and Reaction Buffer (Thermo Fisher Scientific), 40U RiboLock
RNase Inhibitor (Thermo Fisher Scientific), 0.4 mM Deoxynucleotide (dNTP)
Solution Mix (New England BioLabs, Ipswich, MA, USA) and 100$\mu$M pentadecamers
(Eurofins Genomics, Ebersberg, Germany). cDNA synthesis included -RT and
non-template controls. The following temperature profile was used for cDNA
synthesis: pre-primer extension (10 min at \SI{25}{\celsius}), DNA
polymerization (50 min at \SI{37}{\celsius}) and enzyme deactivation (15 min at
\SI{70}{\celsius}).

\subsection{TaqMan card design and measurement}

For gene expression analysis, Custom TaqMan\texttrademark Gene Expression Array
Cards format 32 (Thermo Fisher Scientific) were designed. Genes of interest
were chosen from previous studies on Antarctic krill \citep[and Höring et al.,
in prep]{piccolin_seasonal_2018} and from the Krill Transcriptome Database
\citep{sales_krilldb:_2017}. For some genes of interest, the primer/probe
design was adopted from the study by \citet{piccolin_seasonal_2018}. For the
other genes, where the primer/probe design was not yet available, annotation
was reviewed by blastn and blastx searches on the web interface of NCBI
(\url{https://blast.ncbi.nlm.nih.gov/Blast.cgi}; \citet{johnson_ncbi_2008}).
ORFfinder (\url{https://www.ncbi.nlm.nih.gov/orffinder/}) was used to identify
the open reading frame (ORF) of the sequences that conformed with the reading
frame and the identified protein domains from the blastx searches. If
applicable, the alignment of seasonal RNAseq data for each sequence
(Publication I) was then used to find an amplicon within the ORF with the least
single nucleotide polymorphisms using the programme Tablet
\citep{milne_tabletnext_2010}. The segments of interest were cut in the
programme \code{ApE v.2.0.53c} (\url{http://jorgensen.biology.utah.edu/wayned/ape/}).
No repeats were found for these parts of the genes
(\url{http://www.repeatmasker.org/cgi-bin/WEBRepeatMasker}).  The sequences
were then submitted to the Custom TaqMan\textsuperscript{\textregistered} Assay
Design tool for automatic primer and probe design for each gene (Thermo Fisher
Scientific,
\url{https://www.thermofisher.com/order/custom-genomic-products/tools/gene-expression/}).
The sequences of primers and probes used in this study are summarized in
supplementary Table SI. 

% TODO: Table SI ???

Gene expression analysis via TaqMan\textsuperscript{\textregistered} Cards was
conducted on a ViiA\texttrademark 7 Real-Time PCR System (Thermo Fisher
Scientific) according to manual instructions. \SI{400}{\nano\gram} cDNA per
sample and TaqMan\textsuperscript{\textregistered} Gene Expression Master Mix
(Thermo Fisher Scientific) with a final loading volume of
\SI{100}{\micro\liter} per sample were used.  Dilution curves were performed
for the analysis of primer efficiency. 

\subsection{Gene expression analysis}

Data quality control and normalisation were carried out using the software
\code{qbase+ 3.2} (Biogazelle, Zwijnaarde, Belgium). The median Cq value was
calculated for each data point from three technical replicates. The preliminary
geNorm analysis \citep{vandesompele_accurate_2002} implemented in \code{qbase+}
suggested two housekeeping genes for data normalisation: \textit{RNA polymerase
I-specific transcription initiation factor RRN3 isoform 1 (rrn3)} and
\textit{ubiquitin carboxyl-terminal hydrolase 46 (usp46)}. These genes have
already been found to show stable gene expression in Antarctic krill in
previous photoperiodic controlled laboratory experiments \citep[; Bisconton et
al., in prep.]{piccolin_seasonal_2018}. The stability of the two reference
genes was validated for the final dataset by calculating the reference target
stability in \code{qbase+} (\textit{rrn3}: M=0.285 \& CV=0.1; \textit{usp46}:
M=0.285 \& CV=0.098). Gene expression data were then normalised using the
method by \citet{hellemans_qbase_2007}, a modified version of the
delta-delta-Ct method that incorporates PCR efficiency correction for each
gene, multiple reference gene normalisation as well as proper error
propagation, implemented in \code{qbase}. The normalised expression values of
the target genes with standard error can be found in supplementary table SII.

%TODO: Table SII???

%TODO below: Figure 1
Statistical analysis and visualisation of gene expression results were
performed in \code{RStudio version 1.1.456} \citep{rstudio_team_rstudio:_2016}.
The heatmap for Fig. 1 was generated in the \code{R} package \code{gplots}
\citep{warnes_gplots:_2016} using the median values per time point and
treatment. To test for rhythmic seasonal expression patterns of the analysed
genes, the \code{R} package \code{rain} \citep{thaben_detecting_2014} was used.

To identify differences in gene expression patterns between light
treatments/conditions (52$^{\circ}$, 66$^{\circ}$S and DD), sparse partial
least squares discriminant analysis (sPLS-DA) was used (\code{R} package
\code{mixOmics} \citep{rohart_mixomics:_2017}). sPLS-DA analysis was conducted
for each month separately (7 subsets). The subsets included krill of both sexes
that were analysed together, because a preliminary Principal Component Analysis
(PCA) revealed that major parts of the variation in the subsets were not
related to sex. sPLS-DA fitted a classifier multivariate model for each subset
while assigning the samples into the three light treatments (52$^{\circ}$,
66$^{\circ}$S and DD). In addition, this method allowed variable selection
including the optimal number of components and features (genes) leading to the
lowest overall error rates of the final model. If the parameter tuning of the
sPLS-DA (5-fold cross-validation, repeated 100 times) indicated an optimal
number of components = 1, we considered using a model with two components for
visualisation purposes. Performance of the final models was assessed by the
overall error rates for each component and the stability of the selected genes.
The parameters chosen for the final models can be found in Table II. Sample
plots, loading plots and clustered image maps from the \code{R} package
\code{mixOmics} were used for the visualisation of the sPLS-DA results.

%TODO Table II above



%----------------------------------------------------------------------------------------
%	RESULTS
%----------------------------------------------------------------------------------------

\section{Results}

\subsection{Analysis of seasonal rhythmicity of gene expression}

Using \code{RAIN}, significant rhythmic patterns of gene expression were
detected in Antarctic krill over a period of 12 months in the three treatments
simulating the low-latitude light regime 52$^{\circ}$S, the high latitude light
regime 66$^{\circ}$S and constant darkness (DD) (Fig.1, Table III).

%TODO above Fig 1 Table III


Under treatment 52$^{\circ}$S, 7 genes had significant seasonal rhythmicity
with functions in glycolysis (\textit{phosphofructokinase-6} alias
\textit{pfk6}), citric acid cycle (\textit{citrate synthase} alias
\textit{cs}),  respiratory chain (\textit{ATP synthase subunit gamma} alias
\textit{atp}), prostaglandin biosynthesis (\textit{carbonyl reductase 1} alias
\textit{cbr1}), lipid transport (\textit{fatty acid binding protein} alias
\textit{fabp}), translation (\textit{ribosomal protein S13} alias
\textit{rps13}), and feeding (\textit{allatostatin C} alias \textit{astc}).

Under treatment 66$^{\circ}$S, we found the highest number of genes displaying
significant rhythmicity throughout the year (15). These genes are involved in
glycolysis (\textit{pfk6} \& \textit{glyceraldehyde-3-phosphate dehydrogenase}
alias \textit{gapdh}), citric acid cycle (\textit{cs}), respiratory chain
(\textit{atp} \& \textit{cytochrome c oxidase subunit NDUFA4} alias
\textit{ndufa4}), lipid transport and metabolism (\textit{fabp} \&
\textit{fatty acid synthase} alias \textit{fasn} \& \textit{galactosylceramide
sulfotransferase} alias \textit{gal3st}), prostaglandin biosynthesis
(\textit{cbr1} \& \textit{hematopoietic prostaglandin D synthase} alias
\textit{hpgds}), activation of neuropeptides (\textit{neuroendocrine convertase
1} alias \textit{nec1}), circadian clock (\textit{timeless1} alias
\textit{tim1}), translation (\textit{rps13} and \textit{ribosomal protein S18}
alias \textit{rps18}), and feeding (\textit{anionic trypsin 1} alias
\textit{trp1}).

Under treatment DD, only two genes showed significant seasonal rhythmicity,
which were involved in energy metabolism (\textit{atp}) and citric acid cycle
(\textit{cs}). 

We did not find significant rhythmicity in any of the treatments for the
following genes: \textit{long-chain-fatty-acid-CoA ligase (acsl),
vitellogenin-like gene (vtlgl), farnesoic acid O-methyltransferase (famet),
clock (clk), cryptochrome 2 (cry2), RNA polymerase II largest subunit RPB1
(rpb1), span-like gene (span),} and \textit{low-density lipoprotein
receptor-related protein 1 (lrp1)} (for function see Table III).

\subsection{Analysis of monthly gene expression signatures using sPLS-DA}

We used sPLS-DA to exploratively assess monthly differences in gene expression
of Antarctic krill in response to the different light treatments (52$^{\circ}$S, 66$^{\circ}$S,
and DD) for the first time. Gene expression data from each month were examined
separately within the following subsets: Apr15 (autumn), Jun15 (early winter),
Aug15 (late winter), Oct15 (spring), Dec15 (early summer), Feb16 (late summer),
and Apr16 (second autumn). Even though we were not able to fully separate the
treatments in most of the subsets (Fig. 2 \& Fig. 3) (likely due to the low
number of replicates per month/treatment), this type of analysis provided first
insights into general gene expression patterns, putatively characterizing the
response to the three different light treatments throughout the year. The gene
sets selected by sPLS-DA for each monthly subset contributed to the separation
of the treatments (Fig. 2-3, details for each subset are outlined below). They
were analysed in terms of loadings for each gene per component, which can be
interpreted as a positive correlation of the gene expression pattern with the
corresponding treatment of the sample plots (i.e. “up-/downregulated”; Fig.
2-3), irrespective of positive or negative loading values. The sPLS-DA results
for each subset are summarized in Tables IV to X including the selected genes
for each component of interest, their respective functions, and their loadings.

% TODO Tables, Figures

For the Apr15 subset (autumn), we focussed on the sPLS-DA results of component
1, as component 2 did not improve the discrimination of the three treatments
(Fig. 2a). Component 1 discriminated between individual krill from the two
light treatments (66$^{\circ}$S and 52$^{\circ}$S). Krill from treatment DD
displayed a high variability in gene expression with similarities to krill
samples of both light treatments (66$^{\circ}$S and 52$^{\circ}$S) and no
distinct pattern. The detected gene signature of component 1 that contributed
to the separation of treatment 66$^{\circ}$S and 52$^{\circ}$S can be found in
Table IV.

For the Jun15 subset (early winter), component 1 mostly contributed to the
separation of individual krill from the light treatments
52$^{\circ}$S/66$^{\circ}$S and treatment DD, with no further discrimination
between the latitudinal light treatments 52$^{\circ}$S and 66$^{\circ}$S, and a
less distinct pattern in individual krill from treatment 66$^{\circ}$S (Fig.
2b). Component 2 of the Jun15 model did not improve the discrimination between
the treatments. The gene expression patterns of component 1 that characterized
the separation between the treatments 52$^{\circ}$S/66$^{\circ}$S and DD are
summarized in Table V.

For the Aug15 subset (late winter), both component 1 and component 2
contributed to the discrimination of the treatments (Fig. 2c). Component 1
distinguished between individual krill from the light treatments
66$^{\circ}$S/52$^{\circ}$S and treatment DD, with a less distinct pattern for
individual krill from treatment 52$^{\circ}$S. The second component contributed
to the distinction between krill from latitudinal light treatments
66$^{\circ}$S and 52$^{\circ}$S, whereas the pattern in krill from treatment DD
was less distinct. The gene signatures for components 1 and 2 and respective
functional information for these genes can be found in  Table VI.

For the Oct15 subset (spring), both component 1 and 2 discriminated between
individual krill from the latitudinal light treatments
66$^{\circ}$S/52$^{\circ}$S and treatment DD (Fig. 2d). For the first component
the pattern was less distinct for individual krill from treatment
52$^{\circ}$S, whereas for the second component the pattern was less clear for
individual krill from treatment 66$^{\circ}$S. The selected gene sets that
characterized the separation between treatments 66$^{\circ}$S/52$^{\circ}$S and
DD in 'spring' are shown in Table VII.

For the Dec15 subset (early summer), component 1 discriminated between
individual krill from treatment DD and the light treatments
66$^{\circ}$S/52$^{\circ}$S, with a less distinct pattern for krill from
treatment 52$^{\circ}$S (Fig. 3a). Component 2 contributed to the separation of
individual krill from treatment DD/66$^{\circ}$S and treatment 52$^{\circ}$S. The
respective gene signatures can be found in Table VIII.

For the Feb16 subset (late summer), component 1 contributed to the separation
of individual krill from treatment DD and the light treatments
66$^{\circ}$S/52$^{\circ}$S, with a less distinct pattern for krill from
treatment 52$^{\circ}$S (Fig. 3b). The second component discriminated between
krill from treatment 52$^{\circ}$S and treatments DD/66$^{\circ}$S. The selected
gene sets for component 1 and 2 are shown in Table IX.

For the Apr16 subset (second autumn), we focussed on component 1, because the
sPLS-DA results for component 2 did not improve the discrimination of the
treatments (Fig. 3c). The first component discriminated between individual
krill from treatment 66$^{\circ}$S and treatments 52$^{\circ}$S/DD. The gene
signature of component 1that contributes to this separation is summarized in
Table X.

Distinct differences in the gene expression patterns between krill from the
latitudinal light treatments 52$^{\circ}$S and 66$^{\circ}$S were found in the
period of late summer (see loadings for component 2 of the Feb16 subset in
Table IX) and autumn (see loadings for component 1 of Apr15 subset in Table IV
\& component 1 of Apr16 subset in Table X). In late summer (Feb16 subset, Table
IX), the positive loadings of component 2 of the sPLS-DA results indicated an
upregulation of genes related to translation and the upregulation of metabolic
genes with functions in the citric acid cycle, the respiratory chain, and lipid
metabolism in individual krill from treatment 66$^{\circ}$S and DD with respect
to krill from treatment 52$^{\circ}$S. For the first and second autumn, we found
similar expression patterns of metabolism-related genes when comparing krill
from the latitudinal light treatments 66$^{\circ}$S and 52$^{\circ}$S. In the
first autumn (Apr15 subset, Table IV), the negative loadings of component 1 of
the selected gene set showed an upregulation of metabolic genes related to
glycolysis, respiratory chain, lipid metabolism, and an upregulation of genes
related to lipid transport in krill from treatment 66$^{\circ}$S with respect to
krill from treatment 52$^{\circ}$S. In the second autumn (Apr16 subset, Table
X), the negative loadings of component 1 of the detected gene signature
indicated an upregulation of genes related to glycolysis, lipid metabolism, and
lipid transport in krill from treatment 66$^{\circ}$S with respect to treatments
52$^{\circ}$S/DD. In the late summer and autumn subsets (Feb16, Apr15, Apr16),
the upregulation of genes related to general metabolism was also accompanied by
the upregulation of the development-related gene \textit{span} and two genes
with putative regulatory roles (\textit{nec1}, \textit{lrp1}), and for the
autumn subsets only the upregulation of the feeding-related gene \textit{trp1}.

In early winter (Jun15 subset) and spring (Oct15 subset), krill from the
latitudinal light treatments 66$^{\circ}$S and 52$^{\circ}$ were not
discriminated by sPLS-DA analysis, and only minor differences were detected in
early summer (Dec15 subset, Table VIII). However, in late winter (Aug15 subset,
Table VI), the negative loadings of component 2 indicated an upregulation of
genes related to the regulatory processes (\textit{nec1}), development
(\textit{span}), and translation and transcription in krill from treatment
52$^{\circ}$S with respect to treatment 66$^{\circ}$S. Moreover, the positive
loadings of the same component showed an upregulation of genes related to lipid
transport in krill from treatment 66$^{\circ}$S with respect to treatment
52$^{\circ}$S.

Under treatment DD, we observed an upregulation of genes related to metabolic
and potential regulatory processes during winter and spring. In early winter
(Jun15 subset, Table V), the selected genes with positive loadings indicated an
upregulation of the lipid metabolic gene \textit{fasn}, and of genes related to
lipid transport and the biosynthesis of methyl farnesoate (\textit{famet}) in
krill from treatment DD with respect to the light treatments
52$^{\circ}$S/66$^{\circ}$S. In late winter (Aug15 subset, Table VI), the
positive loadings of component 1 showed an upregulation of metabolic genes
related to glycolysis, citric acid cycle and the respiratory chain, and the
regulatory gene \textit{nec1} in individual krill of treatment DD with respect
to the light treatments 52$^{\circ}$S/66$^{\circ}$S. In spring (Oct15 subset,
Table VII), the positive loadings of component 1 indicated an upregulation of
genes related to glycolysis, citric acid cycle, respiratory chain, translation
and transcription, and of genes with putative regulatory functions
(\textit{nec1}, \textit{famet}, \textit{lrp1}) in krill from treatment DD with
respect to treatments 66$^{\circ}$S/52$^{\circ}$S.

The sPLS-DA results indicate that krill from treatment DD showed a
downregulation of most genes during summer with respect to the light
treatments. In early summer (Dec15 subset, Table VIII), the identified gene
signature of component 1 points to an upregulation of the putative regulatory
gene \textit{famet} and the down-regulation of the majority of the other
selected genes (amongst others metabolic genes) in krill from treatment DD with
respect to krill from the light treatments 66$^{\circ}$S/52$^{\circ}$S. In late
summer (Feb16 subset, Table IX), component 1 solely indicated an upregulation
of the glycolysis-related gene \textit{gapdh} in krill from treatment DD (with
respect to the light treatments 66$^{\circ}$S/52$^{\circ}$S), whereas the other
selected genes e.g. related to lipid metabolism, lipid transport were
down-regulated.

For clock gene expression and prostaglandin metabolism, we observed similar
expression patterns with an upregulation in the light treatments
52$^{\circ}$S/66$^{\circ}$S and a downregulation under constant darkness. This
was especially evident from the sPLS-DA analysis of the subsets Jun15 (early
winter, Table V), Oct15 (spring, Table VI), Dec15 (early summer, Table VIII),
and Feb16 (late summer, Table IX) that all revealed a discrimination between
krill from the light treatments 52$^{\circ}$S/66$^{\circ}$S and constant
darkness. The detected gene signatures of these subsets suggested a mostly
complete upregulation of genes related to the circadian clock and prostaglandin
metabolism in krill from the light treatments 52$^{\circ}$S/66$^{\circ}$S with
respect to treatment DD. A 'co-upregulation' of clock genes and prostaglandin
metabolic genes was also indicated in autumn, with an upregulation of these
genes in individual krill from treatment 52$^{\circ}$S with respect to treatment
66$^{\circ}$S during the first autumn (Apr15 subset, Table IV), and a
upregulation of these genes in individual krill from 66$^{\circ}$S with respect
to treatment 52$^{\circ}$S/DD in the second autumn (Apr16 subset, Table X). In
late winter (Aug15 subset, Table VI), genes related to circadian clock and
prostaglandin metabolism were not selected for the first component which
contributed to the separation of krill from the light treatments
66$^{\circ}$S/52$^{\circ}$S and treatment DD, and for the second component, these
genes showed partly upregulation in both individual krill from treatment
52$^{\circ}$S and 66$^{\circ}$S.

We did not find fully conclusive seasonal expression patterns for the feeding
indicator genes astc and trp1 for most of the sPLS-DA results. For instance for
the late winter subset (Aug15, Table VI), component 2 indicated the
upregulation of the gene astc (putative feeding inhibitor) and the
downregulation of  trp1 (putative digestive enzyme) in krill from treatment
52$^{\circ}$S with respect to treatment 66$^{\circ}$S (where we would expect the
opposite). Therefore, we did not focus on these genes for our main conclusions. 


\section{Discussion}

This study analysed the seasonal gene expression patterns of Antarctic krill in
a long-term laboratory experiment at simulated low-latitude light regime
52$^{\circ}$S, at high-latitude light regime 66$^{\circ}$S and at constant
darkness over two years. It complemented the earlier study by
\citet{horing_light_2018} that analysed morphometric and lipid content data
from the same experiment.

The simulated light regimes 52$^{\circ}$S and 66$^{\circ}$S seem to induce
seasonally rhythmic gene expression patterns that are more pronounced under the
high latitude light regime (66$^{\circ}$S). The rhythmically expressed genes
have functional roles in the processes of glycolysis, citric acid cycle,
respiratory chain, lipid metabolism, lipid transport, prostaglandin
biosynthesis, the activation of neuropeptides, the circadian clock,
translation, and feeding. The predominant loss of rhythmic gene expression
under constant darkness further suggests that photoperiodic cues are required
to maintain the seasonal gene expression patterns of Antarctic krill over
longer periods.

The analyses of monthly gene expression patterns of our laboratory experiment
further indicate that different latitudinal light regimes may trigger flexible
gene expression patterns in Antarctic krill. In late summer and autumn, we
found an upregulation of genes related to different metabolic processes, lipid
metabolism, transport in Antarctic krill kept under the high-latitude light
regime 66$^{\circ}$S with respect to krill from the low-latitude light regime,
which may be an adaptive mechanism to facilitate enhanced lipid accumulation at
higher latitudes in this period of the year. Antarctic krill that is affected
by a low latitude light regime may also prepare earlier for spring, which is
suggested by our observation of an upregulation of genes related to the
activation of neuropeptides, development, translation and transcription in
krill kept under the low-latitude light regime 52$^{\circ}$S with respect to
krill kept under the high-latitude light regime 66$^{\circ}$S in late winter.

Under constant darkness, Antarctic krill does not seem to enter a strong state
of metabolic depression, based on the observation of the upregulation of genes
related to different metabolic and regulatory processes with respect to the
light treatments during winter and spring. The observed downregulation of most
genes during summer in Antarctic krill kept under constant darkness may be
explained by the missing \textit{Zeitgeber} that may have caused an arrhythmic
condition in these krill.

The prostaglandin metabolism related genes \textit{hpgds} and \textit{crb1} as
well as the putative regulatory genes \textit{famet nec1}, and \textit{lrp1}
may have important roles in the regulation of the seasonal reproductive cycle
and metabolism of Antarctic krill. Constant darkness seems to lead to a
disruption of the prostaglandin metabolism and an upregulation of
\textit{famet}, \textit{nec1} and \textit{lrp1} during winter and spring which
coincide with the observation of a less pronounced reproductive cycle under
constant darkness \citep{horing_light_2018} and the enhanced expression of
metabolic genes in winter. Moreover, a link between clock gene expression and
the expression of genes related to prostaglandin biosynthesis may be suggested
by the finding of similar expression patterns in these genes. Both simulated
latitudinal light regimes seem to lead to an upregulation of genes related to
the circadian clock and prostaglandin biosynthesis, whereas constant darkness
seems to trigger a downregulation of these processes. 

The genes \textit{rps13, rps18, rpb1, usp46} and \textit{rrn3} were originally
tested as housekeeper candidates, because they proved to have a stable
expression under variable photoperiods and seasonal conditions in previous
studies \citep[; and Biscontin et al., in prep]{hafker_calanus_2018,
piccolin_photoperiodic_2018, piccolin_seasonal_2018, shi_distinct_2016}. Based
on geNorm analysis and the observation of partly seasonally rhythmic expression
patterns, we cannot confirm the stability of \textit{rps13}, \textit{rps18} and
\textit{rpb1} in photoperiodic-controlled long-term laboratory experiments with
Antarctic krill. Our data rather suggests that \textit{usp46} and \textit{rrn3}
are suitable reference genes when studying gene expression of Antarctic krill
under different simulated light regimes over longer periods. 

The observed expression patterns of the feeding related genes \textit{astc} and
\textit{trp1} may only partly explain the seasonal pattern of feeding that was
derived from a morphometric index measured in the same experiment
\citep{horing_light_2018}. \citet{horing_light_2018} observed a steady increase
in feeding index with a certain seasonality showing a peak of feeding in late
autumn/winter under the three different light treatments (52$^{\circ}$S,
66$^{\circ}$S and DD). The gene \textit{trp1} coding for a digestive enzyme
shows a significant rhythmic pattern under the high latitude light regime
66$^{\circ}$S (but not significant for treatment 52$^{\circ}$S and DD) in this
study. The lower expression of \textit{trp1} during winter and spring and its
higher expression during summer and autumn may indicate a seasonal cycle of
digestive activity that may be mediated by the photoperiod-dependent feeding
activity \citep{teschke_simulated_2007} and thereby leads to a peak in feeding
index in late autumn \citep{horing_light_2018}. However, trypsin may also be
involved in the degradation of cuticle during the moult cycle of Antarctic
krill \citep{seear_differential_2010}.

The potential feeding inhibitor \textit{astc} showed a more irregular pattern
in our study where a significant rhythmic pattern was solely observed under
treatment 52$^{\circ}$S. For instance, the high expression values of
\textit{astc} under the low latitude light regime 52$^{\circ}$S with respect to
66$^{\circ}$S in late winter cannot be explained by the simulated light
conditions, because one would rather expect a higher feeding inhibition in
winter under the high latitude light regime. This may likely be explained by
the multiple functional roles of \textit{astc} which may have introduced
additional variation in our dataset. In crustaceans, C-AST like peptides were
found to have modulatory roles in feeding and locomotion
\citep{wilson_distribution_2010} and cardioactive activity
\citep{dickinson_identification_2009}. In insects, allatostatins have been
shown to inhibit the production of juvenile hormone, an important regulator of
growth, development and reproduction \citep{weaver_neuropeptide_2009}. 

The gene expression results under the latitudinal light treatment 66$^{\circ}$S
and constant darkness are generally comparable to the study by
\citet{piccolin_seasonal_2018} who investigated the seasonal metabolic cycle of
Antarctic krill in a 1-year laboratory experiment. However, the metabolic genes
\textit{cs}, \textit{pfk6} and \textit{atp} had their lowest gene expression
values in late winter (August) in the current study, whereas
\citet{piccolin_seasonal_2018} has already found an increase of expression of
these genes under treatment 66$^{\circ}$S in the same month. Moreover, we did
not find a peak expression of the clock gene \textit{cry2} in February, but the
other two clock genes \textit{tim1} and \textit{clock} revealed similar
patterns as in the study by \citet{piccolin_seasonal_2018}. These differences
may be caused by the slight differences in the simulation conditions of the
latitudinal light regime 66$^{\circ}$S as well as the different sampling
conditions of Antarctic krill in both studies. \citet{piccolin_seasonal_2018}
collected the Antarctic krill over a 24h period (instead of sampling at one
time point of the day) and used much higher replicate numbers for each sampling
month than in the current study.

The incomplete discrimination between the simulated latitudinal light regimes
52$^{\circ}$S, 66$^{\circ}$S and constant darkness in this study may on the one
hand represent a genetic or phenotypic variability in the population of
Antarctic krill that was used for the photoperiodic long-term experiments. On
the other hand, the multivariate statistical approach sPLS-DA used in this
study was originally developed for high throughput biological datasets
\citep{le_cao_sparse_2011}.

Thus, our dataset might be too small, the gene expression patterns between the
treatments too similar, or it may not contain the genes that are most relevant
to discriminate between the treatments. Due to the limited number of Antarctic
krill samples available from the long-term experiment, we decided to analyse
krill of mixed sex, size and moult stages, which may have introduced further
variability in the dataset. Moreover, higher replicate numbers ($\leq$10 krill
samples per month and treatment) may further improve the gene expression
results in future long-term experiments with Antarctic krill.

When comparing the results from this long-term experiment to observations in
Antarctic krill in the field, we are able to draw conclusions regarding the
possible control mechanisms of the different seasonal processes in Antarctic
krill.

Most of our results show strong similarities to observations from the field
which indicate that the seasonal cycle of Antarctic krill and the underlying
gene expression patterns seem to be mainly regulated by light regime. Under the
photoperiodic controlled laboratory conditions applied here, an upregulation of
genes with various metabolic and regulatory functions was found during summer.
This observation agrees with findings for Antarctic krill in the field that
were sampled in different latitudinal regions in the Southwest Atlantic sector
of the Southern Ocean \citep[and Publication I]{seear_seasonal_2012}. Moreover,
the repressed gene expression levels suggest that Antarctic krill enters a
state of winter quiescence under the photoperiodic controlled laboratory
conditions and constant food supply. Therefore, we propose that the observed
winter quiescence in Antarctic krill in the field \citep{meyer_seasonal_2010}
may be mainly controlled by light regime.

The field study by Höring et al. (Publication I) formed the basis for the
selection of several genes in this study that were found to be upregulated in
Antarctic krill from the field during summer. These genes included the hormone
metabolism related genes \textit{hpgds}, \textit{cbr1} and \textit{nec1}, the
lipid transport related genes \textit{vtlgl} and \textit{fabp}, the lipid
metabolism related genes \textit{fasn}, \textit{acsl} and \textit{gal3st}, the
feeding related gene \textit{trp1}, the development related gene \textit{span},
the energy metabolism related gene \textit{ndufa4}, the carbohydrate metabolism
related gene \textit{gapdh}, and the receptor related gene \textit{lrp1}. Most
of these genes showed significantly seasonal patterns of gene expression under
the simulated light regimes in this study, which suggests that their seasonal
expression patterns may also be regulated by light regime in the field.

For the first time, we show that Antarctic krill is able to adjust its gene
expression to different latitudinal light regimes in the Southern Ocean. In
late summer and autumn, we found an upregulation of genes mostly involved in
carbohydrate and energy metabolism, lipid metabolism and lipid transport under
the simulated light regime 66$^{\circ}$S. The lipid metabolic gene \textit{fasn}
plays a role in fatty acid synthesis for storage
\citep{chirala_structure_2004}, whereas the gene \textit{acsl} codes for an
enzyme that activates long-chain fatty acids for both synthesis and degradation
of lipids \citep{mashek_long-chain_2007}. The upregulation of these processes
in late summer and autumn under the simulated light regime 66$^{\circ}$S (with
respect to 52$^{\circ}$S) may be explained by an enhanced lipid accumulation in
Antarctic krill in this period that is required under the high-latitude light
regime in preparation for winter. This is also found in the field where higher
lipid stores were observed in the Lazarev Sea region (light conditions similar
to light regime 66$^{\circ}$S) with respect to the low-latitude region South
Georgia where prolonged photoperiods and no sea ice cover occur during winter
(light conditions similar to light regime 52$^{\circ}$S)
\citep{schmidt_feeding_2014}. These results support the assumption that the
seasonal lipid metabolic cycle of krill is regulated by latitudinal light
regime \citep{horing_light_2018}. 

Another indication for the variable effect of different latitudinal light
regimes on Antarctic krill is the observation of the more pronounced seasonal
cycle of gene expression that was found under the high latitude light regime
66$^{\circ}$S in this study indicated by the highest number of significantly
rhythmic genes compared to the other treatments.  Moreover, we found an
upregulation of genes related to the activation of neuropeptides, development,
translation and transcription under the simulated light regime 52$^{\circ}$S
(with respect to 66$^{\circ}$S) in late winter, which may indicate that the
low-latitude light regime triggers an earlier preparation for spring in these
krill, and consequently a shorter/or less pronounced winter quiescence at lower
latitudes. Similar effects have already been observed in the field by Höring et
al. (Publication I) who found less seasonal differences in gene expression between
summer and winter krill from the low-latitude region South Georgia compared
with regions at higher latitudes.

However, some observations from the field cannot be confirmed in this study. In
the low-latitude region South Georgia, a higher feeding activity and an
enhanced expression of feeding related genes was observed with respect to
regions at higher latitudes in winter \citep{schmidt_feeding_2014,
seear_seasonal_2012}. Although we found a seasonal pattern of feeding in our
photoperiodic controlled long-term experiment, we were not able to detect a
higher feeding activity under the low-latitude light regime 52$^{\circ}$S in
winter. These findings may support the idea that the seasonal feeding behaviour
of Antarctic krill may be regionally adjusted in the field by other
environmental cues than light regime such as regional food supply
\citep{schmidt_feeding_2014}. Regional feeding conditions may also explain the
complex gene expression patterns that were observed in Antarctic krill in
different latitudinal regions by Höring et al. (Publication I).

Moreover, this study gives insight into different signalling pathways that seem
to be involved in the regulation of the seasonal cycle of Antarctic krill. This
study suggests that seasonal clock gene expression is regulated by light
regime. We found partly seasonal rhythmicity of clock gene expression and a
general upregulation of clock genes under the simulated light regimes
52$^{\circ}$S and 66$^{\circ}$S throughout the entire study period, whereas clock
gene expression was disrupted in Antarctic krill under constant darkness due to
the missing \textit{Zeitgeber}. \citep{piccolin_seasonal_2018} proposed that
seasonal differences in clock gene expression may be linked to the regulation
of seasonal life cycle events in Antarctic krill. Clock genes may play a role
in the seasonal time measurement in Antarctic krill, because a similar
regulation mechanism was found to control diapause in insects
\citep{meuti_functional_2015}.  However, the functional link between clock gene
expression and the seasonal output pathways still needs to be validated in
knock-out experiments with Antarctic krill.

Another level of regulation includes prostaglandins which are physiologically
active lipid compounds that may control the reproductive cycle of Antarctic
krill. Prostaglandins regulate the ovarian development
\citep{wimuttisuk_insights_2013}, gonad maturation and moulting in crustaceans
\citep{nagaraju_reproductive_2011}. For the prostaglandin biosynthesis related
genes \textit{hpgds} and \textit{cbr1} \citep{wimuttisuk_insights_2013}, we
found a similar pattern as in the clock genes with a disruption of the seasonal
rhythmic expression patterns of these genes in Antarctic krill under constant
darkness. Based on the observation that Antarctic krill's reproductive cycle
was weakened under constant darkness \citep{horing_light_2018}, we suggest that
prostaglandins are mainly involved in the regulation of the seasonal cycle of
reproduction in Antarctic krill. Moreover, seasonal patterns of prostaglandin
biosynthesis are regulated by light regime, and possibly linked to clock gene
expression.

We further propose that the gene \textit{famet} may play a role in the
regulation of reproduction, moulting and the initiation of winter quiescence in
Antarctic krill. The gene \textit{famet} codes for an enzyme that catalyses the
final step of the synthesis of methyl farnesoate, a precursor of the insect
juvenile hormone III \citep{gunawardene_function_2002}. Methyl farnesoate
stimulates moulting and reproductive development in crustaceans
\citep{reddy_involvement_2004}. In insects, the suppression of the related
molecule juvenile hormone plays a role in the initiation of diapause
\citep{liu_absence_2017}. Under constant darkness, we detected an upregulation
of \textit{famet} in early winter, spring, and early summer. This may be
explained by the inhibitory mechanism that controls the secretion of methyl
farnesoate from the mandibular organ which is controlled by the eyestalk
neuropeptide mandibular organ-inhibiting hormone in crustaceans
\citep{swetha_reproductive_2011}. Eye stalk ablation leads to increased levels
in methyl farnesoate in the mandibular organ and hemolymph
\citep{tsukimura_regulation_1992}. This may explain our observation of
increased \textit{famet} levels and the disruption of Antarctic krill's
reproductive cycle under constant darkness. Constant darkness probably
interrupts the inhibitory mechanism that controls the secretion from the
mandibular organ which leads to higher synthesis levels and secretion of methyl
farnesoate and therefore advanced maturation stages of Antarctic krill. 

This study also indicates that enzymes related to the family of subtilisin-like
proprotein convertases may play a role in the regulation of seasonal hormone
levels in Antarctic krill. These enzymes are known to activate precursors of
hormones and neuropeptides via proteolysis \citep{zhou_proteolytic_1999}. They
comprise the prohormone convertase 1 (gene \textit{nec1}), which was found to
be involved in the reproduction process of the abalone
\citep{zhou_next-generation_2010}. In crustaceans, prohormone convertase 2 like
genes were found to be expressed in the neuroendocrine cells of the eyestalk
\citep{toullec_molecular_2002} and to bind the crustacean hyperglygemic hormone
\citep{tangprasittipap_structure_2012}. Therefore, they could be involved in
the activation of many neuropeptides that are produced in the neuroendocrine
cells of the eyestalk containing the X-organ-sinus gland complex in crustaceans
\citep{tangprasittipap_structure_2012, toullec_molecular_2002}. These
neuropeptides include the crustacean hyperglemic hormone, the gonad
(vitellogenic) inbiting factor, and the molt inhibiting hormone which have
various functions in the regulation of glucose level, reproduction and growth
in crustaceans \citep{nagaraju_reproductive_2011}. In our study, the prohormone
convertase related gene \textit{nec1} showed a significant seasonal pattern of
expression under the simulated light regime 66$^{\circ}$S with a downregulation
during the winter months. In contrast, \textit{nec1} was found to be
upregulated during late winter and spring in Antarctic krill under constant
darkness. These observations suggest that the gene \textit{nec1} is influenced
by light regime and that its seasonal pattern is disrupted under constant
darkness. This may subsequently lead to a disturbance of the hormonal control
mechanisms of seasonal processes in Antarctic krill such as reproduction or
metabolism. 

Moreover, the cell-surface receptor related gene \textit{lrp1} may be involved
in the regulation of metabolic processes in Antarctic krill. We observed an
upregulation of \textit{lrp1} (and \textit{nec1}) in krill under the
latitudinal light regime 66$^{\circ}$S in late summer and autumn, and under
constant darkness in spring, which was always accompanied with an upregulation
of different metabolic genes. The low-density lipoprotein receptor-related
protein 1 (gene \textit{lpr1}) may have multiple functions, for instance in
cellular lipid homeostasis and the regulation of signalling pathways
\citep{franchini_low-density_2011}.

This study provides further evidence for the effect of different latitudinal
light regimes on the seasonal cycle of Antarctic krill and may therefore be
relevant for future modelling studies of \textit{E. superba} in the Southern
Ocean \citep{horing_light_2018}. Our findings point to a highly flexible
seasonal timing system in Antarctic krill that is highly advantageous under the
extreme latitudinal light regimes in the Southern Ocean, especially under the
observed southward shift of Antarctic krill in the Southwest Atlantic Sector
due to effects by global warming \citep{atkinson_krill_2019}. However, more
research is needed to understand if its seasonal timing system is flexible
enough to respond to climate-induced changes in the timing of phytoplankton
blooms that may lead to 'mismatches' in the energy requirements of Antarctic
krill \citep{durant_climate_2007}. 

Moreover, we give novel insights in the molecular mechanisms that govern the
seasonal cycle of Antarctic krill under the influence of different latitudinal
light regimes. These findings form a basis for future functional studies of the
seasonal timing system in Antarctic krill, by applying e.g. the gene silencing
technique RNA interference \citep{hannon_rna_2002}. If the genome sequence of
\textit{E. superba} becomes available and stable breeding conditions are
established under laboratory conditions in the future, the genome editing
technique CRISPR/Cas9 \citep{sander_crispr-cas_2014} may also be considered for
further functional analyses of genes related to the seasonal cycle of Antarctic
krill.

\section{Conclusion}

We analysed the expression patterns of multiple genes with metabolic and
regulatory functions in Antarctic krill, kept in a controlled two-year
laboratory experiment under simulated latitudinal light regimes of 52$^{\circ}$S
and 66$^{\circ}$S and under constant darkness. On gene expression level, we show
that Antarctic krill responds differently to latitudinal light regimes with a
stronger seasonality observed under the more extreme high-latitude light regime
66$^{\circ}$S. On the contrary, constant darkness seems to disrupt the seasonal
expression patterns of different metabolic and regulatory genes, eventually
affecting Antarctic krill's seasonal cycle of physiology and development over
longer periods. These results suggest that latitudinal light regime is an
important \textit{Zeitgeber} for Antarctic krill and may be the prominent
environmental factor contributing to Antarctic krill's flexible seasonal
response in the different latitudinal habitats of the Southern Ocean. Moreover,
our findings reveal that the circadian clock genes, the physiologically active
compounds prostaglandins and methyl farnesoate as well as the
neuropeptide-processing enzyme prohormone convertase 1 and the low-density
lipoprotein receptor-related protein 1 may contribute to the regulation of
seasonal processes in Antarctic krill such as reproduction, growth and
metabolism.

\section{Acknowledgements}

The staff at the Australian Antarctic Division are acknowledged for their help
during the implementation of the long-term experiment. Sincere thanks go to K.
Oetjen and L. Pitzschler for their support during lab work at the
Alfred-Wegener-Institute.

Funding for this study was made available by the Helmholtz Virtual Institute
“PolarTime” (VH-VI-500: Biological timing in a changing marine environment —
clocks and rhythms in polar pelagic organisms), the ministry of science and
culture (MWK) of Lower Saxony, Germany (Research Training Group
“Interdisciplinary approach to functional bio- diversity research” (IBR)), the
Australian Antarctic Program Project \#4037, and the PACES (Polar Regions and
Coasts in a changing Earth System) programme (Topic 1, WP 5) of the Helmholtz
Association.

\printbibliography[heading=subbibliography]
